docker/vue-app/src/form_editor/components/dialog_content/IndicatorEditingDialog.js

```diff
@@ -161,8 +162,17 @@ export default {
             return this.parentID === null ? 'Section Heading' : 'Field Name';
         },
         showFormatSelect() {
-            //not a header, or in advanced mode, or the format of the header is already a format other than none
-            return this.parentID !== null || this.advancedMode === true || this.format !== '' || hasDevConsoleAccess;
+            // Hide format select ONLY when inWorkflow is true (not stepInWorkflow)
+            if (this.indicatorID && this.indicatorsInWorkflow[this.indicatorID]) {
+                const workflowStatus = this.indicatorsInWorkflow[this.indicatorID];
+                if (workflowStatus.inWorkflow === true) {
+                    console.log('format select hidden due to inWorkflow');

```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/console this might be console.warn, I know Michael Gao has brought this up in the past with someone else's prs. There are some other logs that might need to be pulled as well

Reply:
I removed it, it's not needed.